src/core/global.c

```diff
@@ -80,6 +80,7 @@
 
 #if defined NN_HAVE_MINGW
 #include <pthread.h>
+#define gmtime_r(ptr_numtime, ptr_strtime) gmtime_s(ptr_strtime, ptr_numtime)
```

Comment:
looks like you can take this line of code out of the #if

Reply:
I took a look into and gmtime_r is thread safe version of gmtime provided by pthread library. After GCC 5.0  this method was removed from from pthread.h because "WIN32 C runtime library had been made thread-safe" (c). This quote from http://www.signal.uu.se/Toolbox/dream/windows/pthread.h. I made a commit where I avoid pthread include and use Windows.h defined gmtime_s. That should be enough